src/js/common/helpers/login-helpers.js

```diff
@@ -39,6 +39,8 @@ export function getUserData(dispatch) {
       dispatch(updateProfileField('status', json.data.attributes.va_profile.status));
       dispatch(updateProfileField('services', json.data.attributes.services));
       dispatch(updateLoggedInStatus(true));
+    } else {
+      dispatch(profileLoadingFinished());
```

Comment:
I could just dispatch a `updateLoggedInStatus(false)` action here instead of creating a separate one, but that felt odd? The logged in status is already false in the current state. I'm open to changing that depending on what other people think.